test/combinations.jl

```diff
@@ -117,17 +124,20 @@ This file includes tests for:
         end # for
 
         iter_diff_methods = (
-            (supports.autoenzyme, AutoEnzyme()),
+            (true, FiniteDifference()),
+            (supports.autoenzyme, AutoEnzyme())
         )
 
-        for (supported, diff_method) in iter_diff_methods
+        for (supported, method) in iter_diff_methods
+            f = testable.integrand
+            geometry = testable.geometry
+            sol = testable.solution
+
             if supported
-                @test integral(
-                    testable.integrand, testable.geometry; diff_method = diff_method)≈testable.solution rtol=rtol
+                @test integral(f, geometry; diff_method = method)≈sol rtol=rtol
                 @test MeshIntegrals.supports_autoenzyme(testable.geometry) == true
```

Comment:
```suggestion
	if method isa AutoEnzyme
                @test MeshIntegrals.supports_autoenzyme(testable.geometry) == true
        end
```
Maybe just something like this?

Reply:
Ooohhhh... yeah, I was just code-blind on this one. The loop is asking questions specifically about Enzyme support, but it's the new `FiniteDifference` iteration that throws an error.

Reply:
I'm actually just going to drop the `supports_autoenzyme` checks here since those are now checked more thoroughly in `test/utils.jl`.

Reply:
That's of course also possible.